aeron-agent/src/test/java/io/aeron/agent/ClusterLoggingAgentTest.java

```diff
@@ -141,9 +142,11 @@ private void testClusterEventsLogging(
         Tests.await(WAIT_LIST::isEmpty);
 
         final Counter state = clusteredMediaDriver.consensusModule().context().electionStateCounter();
+
+        final Supplier<String> message = () -> ElectionState.get(state).toString();
         while (ElectionState.CLOSED != ElectionState.get(state))
         {
-            Tests.sleep(1);
+            Tests.wait(Tests.SLEEP_1_MS, message);
```

Comment:
Should call is sleep and have message overload.

Reply:
Done, but I think we should look at removing the `Tests.sleep(long)` method.  We should always be providing context on what is being waited for and the current state on failure.  I'll pick this up as a background task to change existing usages.